src/main.rs

```diff
@@ -118,9 +127,10 @@ enum Commands {
         /// You can only supply version numbers or a custom branch, not both.
         #[arg(long)]
         safe_version: Option<String>,
-        /// The features to enable on the safenode binary.
+        /// The features to enable on the safenode binary. If not provided, the default feature set specified for
```

Comment:
Sorry, just a tiny little pedantic thing: can you please use a line break to keep the documentation style consistent with the other args?

```
/// The features to enable on the safenode binary.
///
/// If not provided, the default feature set specified for the safenode binary are used.
///
/// The features argument is mutually exclusive with the --safenode-version argument.
```